From 97e891607cd1d0a39c00c023f5dd7132a1b77ef6 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Wed, 17 Aug 2016 17:19:40 +0800
Subject: [PATCH 0429/1345] fix: dvfs: a3700: correct the core clock

commit  ed17a09795a69d52a239b28c0bff0e875624e3e0 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

The DVFS uses the CPU core clock, whose core clock index
is 4 in DTS instead of 0.
JIRA: SYSTEMSW-2842

Change-Id: I5d67a3d489aaf31d870c57d5bfdfbbf7de12bd58
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31997
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 14be93c..0f680df 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -241,7 +241,7 @@
 				reg = <0x14000 0x1000>;
 				interrupts = <GIC_SPI 128 IRQ_TYPE_LEVEL_HIGH>;
 				clock-latency = <1000>;
-				clocks = <&core_clk 0>;
+				clocks = <&core_clk 4>;
 			};
 
 			eth0: ethernet@30000 {
-- 
1.7.9.5

